examples/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 function(add_cli_exe T)
-  add_executable(${T} ${ARGN} ${CLI11_headers})
+  add_executable(${T} ${ARGN})
```

Comment:
By removing the headers, doesn't this mean you can't access these in an IDE anymore? (Though if you are using the build-ahead mode, you can - so maybe that's why?)

Reply:
In an IDE, the files are now added to the CLI11 Project, which is new.  They are added as build in precompile mode or with newer CMakes they are added as private sources in the interface library.